### PR TITLE
remove package lock

### PR DESCRIPTION
**Pull request by xiaoyuhen**
use yarn instead of npm

### CODE REVIEW

**Comment by JimLiu** [comment]
Need to add it to gitignore

**Comment by mouyong** [comment]
yarn.lock ?

**Comment by xiaoyuhen** [comment]
@JimLiu Already added to gitignore

**Comment by mouyong** [comment]
@xiaoyuhen you remove package lock but not remove yarn lock?

**Comment by JimLiu** [comment]
haha, personally I prefer npm instead of yarn, but I'm ok with yarn

**Comment by mouyong** [comment]
@xiaoyuhen 部分开发者使用 npm 做为包管理工具，部分开发者使用 yarn 做为包管理工具，当前 PR 移除了 npm 生成的 package-lock.json 的包锁定而未移除 yarn 生成的 yarn.lock 的包锁定。这样会导致一些情况下 两部分开发者使用的包版本不一致。如果要保证一致，需要将 2 个锁文件都进行移除。我是这么看的。你觉得呢？

**Comment by JimLiu** [comment]
Let's keep package-lock.json and remove yarn.lock then. Everyone has npm, but not everyone has yarn.


> @mouyong We should not remove the yarn.lock.
Yarn and npm are both excellent, but we only need one.
> video-react use yarn and yarn.lock describes the last-known-good configuration for a given application.

> @xiaoyuhen 部分开发者使用 npm 做为包管理工具，部分开发者使用 yarn 做为包管理工具，当前 PR 移除了 npm 生成的 package-lock.json 的包锁定而未移除 yarn 生成的 yarn.lock 的包锁定。这样会导致一些情况下 两部分开发者使用的包版本不一致。如果要保证一致，需要将 2 个锁文件都进行移除。我是这么看的。你觉得呢？

**Comment by mondaychen** [comment]
@xiaoyuhen No. Lock file is useful to keep everyone's package versions consistent. But having multiple lock files will cause trouble. So we should have one and only lock file.

Also please use English.

**Comment by mouyong** [comment]
@JimLiu Yep, you are right. thank you tell me these. its very nice.

**Comment by mouyong** [comment]
> @xiaoyuhen No. Lock file is useful to keep everyone's package versions consistent. But having multiple lock files will cause trouble. So we should have one and only lock file.
> 
> Also please use English.

yeah. keep package-lock.json and remove yarn.lock ?

File: .gitignore [hunk 1]
```diff
@@ -55,3 +55,6 @@ dist
 build
 .next
 example/out
+
+# locakfiles
```
Comment (mouyong) [inline .gitignore]: typo. "locakfiles" ？

**Comment by mondaychen** [comment]
Although I use yarn more, to my knowledge I think npm 5 is as good as yarn right now. If there's no more strong objection/argument in 24 hours, let's keep package-lock.json and remove yarn.lock. Sounds good?

**Comment by xiaoyuhen** [comment]
@mondaychen Both yarn and npm are good choices.
I will close and make some changes in the next pr.

**Pull request closed without merging**